test/e2e/curated_packages_test.go

```diff
@@ -209,17 +214,19 @@ func TestCPackagesDockerUbuntuKubernetes121SimpleFlow(t *testing.T) {
 }
 
 func TestCPackagesDockerUbuntuKubernetes122SimpleFlow(t *testing.T) {
+	framework.CheckCuratedPackagesCredentials(t)
 	test := framework.NewClusterE2ETest(t,
 		framework.NewDocker(t),
 		framework.WithClusterFiller(api.WithKubernetesVersion(v1alpha1.Kube122)),
-		framework.WithPackageConfig(t, packageBundleURI(v1alpha1.Kube122),
+		framework.WithPackageConfig(t, packageBundleURI(v1alpha1.Kube121),
```

Comment:
Similar to above. Why was this changed from Kube122 to Kube121 when the test is for Kube122?

Reply:
copy paste mistakes, thanks for catching

Reply:
fixed